docs/application/dotnet/guides/nui/text.md

```diff
@@ -417,7 +449,30 @@ The following example illustrates text alignment:
 field.HorizontalAlignment = HorizontalAlignment.Begin;
 ```
 
-### Use Input Properties in TextField
+### Use HiddenInputSettings property in TextField
+
+The `TextField` class can substitute the input text using the `HiddenInputSettings` property.
+
+To set the `HiddenInputSettings` for detailed options, you can use [HiddenInputProperty](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.HiddenInputProperty.html) and [HiddenInputModeType](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.HiddenInputModeType.html).

```

Comment:
Hello, @annie-samsung 

As I commented on messenger, would you please check the links to API here?

.NET guide are updated by https://github.com/Samsung/tizen-docs/pull/1445,
and one of the update is changing the links to API.

So I think these links should be checked with author of https://github.com/Samsung/tizen-docs/pull/1445
whether if it is ok to use https://samsung.github.io/ link.

Or, another temporary solution in my thought is,
removing the links to `HiddenInputProperty` and `HiddenInputModeType` for now, and adding it back later.


Would you please check?

Reply:
@mijin85cho , sure will sort this out

Reply:
@piotrekgorny, in this PR you can see that in text.md file at line 456 github.io links are provided, could you suggest how do we go about with this PR, do we remove them, an alternate link is available?

@wonrst, Piotr's team just finished a big exercise of removing github.io links from all NUI guides in PR
https://github.com/Samsung/tizen-docs/pull/1387, so adding it back via this PR can cause problem.